eth2near/eth2near-block-relay-rs/Cargo.toml

```diff
@@ -28,6 +28,7 @@ eth2-utility  = { path = "../../contracts/near/eth2-utility" }
 finality-update-verify = { path = "../finality-update-verify" }
 atomic_refcell = "0.1.8"
 bitvec = "*"
+eth2_ssz = { git = "https://github.com/aurora-is-near/lighthouse.git", rev = "b624c3f0d3c5bc9ea46faa14c9cb2d90ee1e1dec" }
```

Comment:
It is better to group crates with the same sources together for readability.

Reply:
Fixed in bee743d